app/controllers/licence_controller.rb

```diff
@@ -61,8 +61,17 @@ def licence_details_from_api_for_local_authority
     licence_details_from_api(snac_from_slug)
   end
 
+  def lgsl
+    content_item_hash["details"]["lgsl_code"]
+  end
+
+  def lgil
+    content_item_hash["details"]["lgil_code"] || content_item_hash["details"]["lgil_override"]
```

Comment:
This `override` logic surprised me, and looks new - could you explain why these changes were necessary (in the commit message)?

Reply:
`lgil_override` is deprecated, so I removed it from here. This line was copied from local_transaction_controller, but there was added years ago.